addons/website_quote/views/website_quotation_backend.xml

```diff
@@ -1,181 +1,179 @@
+        <field name="name">Quotation Templates</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">sale.quote.template</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Click here to create your template.
+            </p><p>
+                Use templates to create polished, professional quotes in minutes.
+                Send these quotes by email and let your customers sign online.
+                Use cross-selling and discounts to push and boost your sales.
+            </p>
+        </field>
+    </record>
 
-        <record id="sale.action_quotations" model="ir.actions.act_window">
-            <field name="help" type="html">
-                <p class="oe_view_nocontent_create">
-                    Click to create a quotation, the first step of a new sale.
-                </p><p>
-                    Odoo will help you handling efficiently the complete sale flow:
-                    from the quotation to the sales order, the
-                    delivery, the invoicing and the payment collection.
-                </p><p>
-                    Promote products with a designed quotation template you send by email.
-                </p><p>
-                    The social feature helps you organizing discussions on each sales
-                    order, and allows your customers to keep track of the sales orders' evolution.
-                </p>
-            </field>
-        </record>
+    <record id="sale.action_quotations" model="ir.actions.act_window">
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Click to create a quotation, the first step of a new sale.
+            </p><p>
+                Odoo will help you handling efficiently the complete sale flow:
+                from the quotation to the sales order, the
+                delivery, the invoicing and the payment collection.
+            </p><p>
+                Promote products with a designed quotation template you send by email.
+            </p><p>
+                The social feature helps you organizing discussions on each sales
+                order, and allows your customers to keep track of the sales orders' evolution.
+            </p>
+        </field>
+    </record>
 
-        <menuitem action="action_sale_quotation_template" id="menu_sale_quote_template" parent="base.menu_sales_config" sequence="1" name="Quotation Templates" groups="base.group_sale_salesman,base.group_sale_manager"/>
-    </data>
-</openerp>
+    <menuitem action="action_sale_quotation_template" id="menu_sale_quote_template" parent="base.menu_sales_config" sequence="1" name="Quotation Templates" groups="base.group_sale_salesman,base.group_sale_manager"/>
+</odoo>
```

Comment:
To minimize the diff, please don't reindent all the code, event if you remove the `data` tags. Thanks !